iotdb-core/datanode/src/main/java/org/apache/iotdb/db/protocol/client/ConfigNodeClient.java

```diff
@@ -374,6 +376,22 @@ private <T> T executeRemoteCallWithRetry(final Operation<T> call, final Predicat
         logger.warn(message, e);
         configLeader = null;
       }
+
+      // If we have detected all configNodes and still not return
+      if (detectedNodeNum >= configNodes.size()) {
+        // Clear count
+        detectedNodeNum = 0;
+        // Wait to start the next try
+        try {
+          Thread.sleep(RETRY_INTERVAL_MS);
```

Comment:
If there are 3 ConfigNodes, then the sleep time will be 10➗3 = 3. Maybe 3 seconds sleep is too short?

Reply:
Perhaps half of confignode's ratis election time is reasonable

Reply:
And we can change the retry num to 15？

Reply:
fixed